src/TileChartMap.jsx

```diff
@@ -0,0 +1,54 @@
+import './TileChartMap.less'
+import { connect } from 'react-redux'
+import React from 'react'
+import { TileMap } from 'cfpb-chart-builder'
+
+export class TileChartMap extends React.Component {
+  componentDidUpdate( prevProps ) {
+    const props = this.props
+    if ( !props.data[0].length ) {
+      return
+    }
+
+    if ( JSON.stringify( prevProps.data ) !== JSON.stringify( props.data ) ) {
+      this._redrawMap()
+    }
```

Comment:
Depending on the size of the data, this could be expensive for a difference check.  Do we have the hashing functions in CCDB?

Reply:
No, doesn't look like we have a hashing function.  I can port it over from explorer.

Reply:
Ok, the underlying code still uses `JSON.stringify`, but would could store the resulting hash in the properties in the future if `.data` becomes too large

Reply:
haha, i literally just moved this over and saw the same thing.  it is a bit cleaner and JSON.stringify is being used all over the place in the componentDidUpdate handler.

I think what the hash function does better is that you have a shorter value stored in memory vs a super long string.  We should convert all of the JSON.stringify to use the hash in a separate PR in one PR in another chore.

Reply:
Agreed!

Reply:
Agreed